asf_core_data/pipeline/mcs/generate_mcs_data.py

```diff
@@ -1,13 +1,22 @@
-#####################################
+# asf_core_data/pipeline/mcs/generate_mcs_data.py
```

Comment:
just a few notes for when we resolve conflicts (as i've also changed this script quite a lot) so that we don't forget.

_Changes that should be kept from this branch:_
- your imports and variables at the top of the file
- function calls get_s3_dir_files(path_to_dir=raw_data_s3_folder) -> check if this has any influence in other files, such as the processing of historical installers and installations;
- function calls save_to_s3() without the s3 argument (this is great btw, I've meant to raise this as an issue) -> this will certainly have an effect on the processing of historical installers, so we need to change that and run the script to see if all is okay.
- all your changes to `generate_processed_mcs_installations()`
- verbose parameter in `generate_and_save_mcs()` and the way you compute the `fully_joined_mcs_epc` variable

All remaining changes should be accepted from dev (but let's do this together in case i've missed something)